src/worker.js

```diff
@@ -118,7 +118,10 @@ function resetPrototype(constructor, attrs) {
 var wasmSourceMapData;
 #endif
 #if USE_OFFSET_CONVERTER
-var wasmOffsetData, wasmOffsetConverter;
+var wasmOffsetData;
+#if MODULARIZE
```

Comment:
I'm confused, what does modularize have to do with the offset converter?

Scanning our ifdefing in other files, I don't see modularize affect `wasmOffsetConverter` (but I didn't read all the places in full detail).

Reply:
`var wasmOffsetConverter;` is defined in `preamble.js` .. i.e. it is defined in the program's JS file and not here in worker.js.    

The way that variable sharing between node, browser, and modularize happens seems to be different.     It has to do with the way the global scope works during `eval` I think.

Here in worker.js we assign to `wasmOffsetConverter` below assuming we are assigning to the variable that is declared in `preamble.js`.   Under node if we *also* declare `wasmOffsetConverter` with `var` here we effectively make separate variable and assign to that, making `wasmOffsetConverter` in the application code remain undefined.

Somehow the double declaration doesn't matter in the browser but does under node.

I'm not sure how `wasmOffsetConverter` get shared in `MODULARIZE`.. bit there was a least one test that failed without this var declared here in `MODULARIZE` mode.  I can't remember which one exactly... I will verify that now.

Reply:
A scoping issue with node.js makes sense. But it's odd to me that it's about modularize. I'd expect this to be `ifdef #USE_OFFSET_CONVERTER`, but maybe I"m missing something.

Reply:
since in preamble there is
```js
#if USE_OFFSET_CONVERTER
var wasmOffsetConverter;
#include "wasm_offset_converter.js"
#endif
```

Reply:
Turns out wasmOffsetConverter was just not working at all with MODULARIZE.  I opened #14755 to fix that.

Reply:
This part of the change is no longer needed.